src/kv/store.h

```diff
@@ -679,8 +679,10 @@ namespace kv
 
       term_of_next_version = t;
       auto h = get_history();
+      LOG_TRACE_FMT("Getting history");
       if (h)
       {
+        LOG_TRACE_FMT("Got history");
```

Comment:
Can we remove this debug logging?

Reply:
Yes

Reply:
I'm tempted to define a logging macro that the TODO detection thing catches, to avoid this happening again.

Reply:
https://github.com/microsoft/CCF/pull/5415